source/Halibut.TestUtils.Contracts/DataStreamExtensionMethods.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.IO;
+
+namespace Halibut.TestUtils.Contracts
+{
+    public static class DataStreamExtensionMethods
```

Comment:
This class would be better suited in `Halibut.Tests`, but it gets used in the `ComplexObjectService` itself so it's easier to keep it here.